src/OrderChocolatCake.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+import RestaurantButton from "./RestaurantButton";
+
+const OrderChocolateCake = () => {
+  //const types=["Pizza","Salad","Chocolate cake"]
```

Comment:
You are not using this component anywhere so the file can be deleted - don't keep unused code🙂